great_expectations/execution_engine/split_and_sample/sqlalchemy_data_splitter.py

```diff
@@ -631,7 +633,7 @@ def _get_params_for_batch_identifiers_from_date_part_splitter(
         Returns:
             List of dicts of the form [{column_name: {date_part_name: date_part_value}}]
         """
-        date_parts: List[DatePart] = self._convert_date_parts(date_parts)
+        date_parts = self._convert_date_parts(date_parts)
```

Comment:
Same comment

Reply:
I'd rather not create new variables as part of this PR.